charts/dbildungs-iam/templates/dbildungs-iam-deployment-bff.yaml

```diff
@@ -1,46 +1,46 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: dbiam-deployment-bff
+  name: {{ include "dbildungs-iam.name" . }}-bff
   labels:
-    app.kubernetes.io/name: dbildungs-iam
+    {{- include "dbildungs-iam.labels" . | nindent 4 }}
 spec:
   selector:
     matchLabels:
-      layer: dbildungs-iam-bff
-  replicas: {{.Values.dbildungsIamReplications}}
+      {{- include "dbildungs-iam.bffLayerLabel" . | nindent 6 }}
+  replicas: {{ .Values.dbildungsIamReplications }}
   template:
     metadata:
-      name: dbildungs-iam
+      name: {{ include "dbildungs-iam.templateName" . }}
       labels:
-        layer: dbildungs-iam-bff
+        {{- include "dbildungs-iam.bffLayerLabel" . | nindent 8 }}
     spec:
       containers:
-        - name: dbildungs-iam-bff
-          image: {{.Values.dbildungsIamContainer}}
+        - name: {{ include "dbildungs-iam.name" . }}-bff
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: Always
           securityContext:
             allowPrivilegeEscalation: false
           command: ["node", "dist/src/backend-for-frontend/main.js"]
           ports:
-             - name: web
-               containerPort: 8080
+            - name: {{ .Values.portName }}
+              containerPort: 8080
           env:
             - name: NODE_ENV
-              value: {{.Values.environment}}
+              value: {{ .Values.environment }}
             - name: DEPLOY_STAGE
-              value: {{.Values.environment}}
+              value: {{ .Values.environment }}
```

Comment:
these two variables just return the same value, whatid NODE_ENV anyway, is DEPLOY_STAGE not sufficient? Would STAGE itself even not be suffiecient?